cfme/control/explorer/policies.py

```diff
@@ -434,6 +421,11 @@ def assign_actions_to_event(self, event, actions):
             view.save_button.click()
         else:
             view.cancel_button.click()
+        # this statement is necessary for test_delete_all_actions_from_compliance_policy
+        view.flash.wait_displayed()
```

Comment:
This is the case for both cancel and save?

Reply:
Without the `wait_displayed()` in there the following statement `check == ...` will not properly evaluate. The flash message does appear for both saving and canceling the event assignment.